app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -115,13 +112,13 @@ class EventDetailsFragment : Fragment() {
             rootView.eventOrganiserName.visibility = View.VISIBLE
             organizerContainer.visibility = View.VISIBLE
 
-
-
             Picasso.get()
-                    .load(event.logoUrl)
-                    .placeholder(AppCompatResources.getDrawable(context!!, R.drawable.ic_person_black_24dp)!!)   //TODO: Make null safe
+                    .load(event.logoUrl).let{it1 ->
+                        ((context?.let { AppCompatResources.getDrawable(it, R.drawable.ic_person_black_24dp) })?.let {it1.placeholder(it)} ?: it1)}
```

Comment:
You can use the Android studio shortcut to wrap it with `?. let call` .
Just remove the exclamation marks and click on the red bulb that appears. This way, the code would look neater.
Please make the change else where too

Reply:
I removed all empty lines, but I have the next PR which addresses all double exclamation points; I will now update that PR  per your recommendations